src/js/active-search.js

```diff
@@ -0,0 +1,12 @@
+import $ from "jquery";
+
+
+const initSearch = () => {
+  $('.js-search-btn').click(function (event) {
+      let x = $('.js-search-btn').parents('.box-bands').addClass('box-bands--active');
```

Comment:
Why do we need the variable?

Reply:
We do not need all `.box-bands` parents. Consider using `.closest('.box-bands')`